joomla/joomla-cms
# Added a new link on the module manager page to add a module for that menutype

Solution for:
http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=27379

This code is actually pretty neat because it allows you to pre-populate any module parameter when creating a new module.  Could have a wealth of other applications.

Test plan:
1. Add a new menu type
2. Check that this link appears in the menu manager list against the new menu "**Add a module for this menu type."
3. Click link and ensure you go to a new menu page and the menutype is populated.  Save and check details.
4. Create other modules and edit other modules and ensure the "menutype" behaves as expected.
